Sources/X509/RelativeDistinguishedName.swift

```diff
@@ -179,3 +193,75 @@ extension RelativeDistinguishedName: DERImplicitlyTaggable {
         }
     }
 }
+
+extension DER {
+    /// Parse the node as an ASN.1 SET OF.
+    ///
+    /// Constructs an array of `T` elements parsed from the set.
+    ///
+    /// - parameters:
+    ///     - of: An optional parameter to express the type to decode.
+    ///     - identifier: The ``ASN1Identifier`` that the SET OF is expected to have.
+    ///     - rootNode: The ``ASN1Node`` to parse
+    /// - returns: An array of elements representing the elements in the sequence.
+    @inlinable
+    public static func lazySet<T: DERParseable>(of: T.Type = T.self, identifier: ASN1Identifier, rootNode: ASN1Node) throws -> some Sequence<Result<T, Error>> {
```

Comment:
It's pretty unfortunate to have to add this, and its supporting code, here: doubly so that we've made it `public`. Can we push this down into ASN1?

Reply:
Yes sure, I just didn't like that particular name and couldn't come up with a better one.

Reply:
I think it's fine.

Reply:
https://github.com/apple/swift-asn1/pull/32